### DISCUSSION TITLE
cn/2023/03/literate-programming-and-reproducible-research/

### DISCUSSION THREAD

**Discussion by giscus** [leovan/leovan.me#36] [Comments]
# cn/2023/03/literate-programming-and-reproducible-research/

文学编程 文学式编程（Literate Programming）是由高德纳提出的编程方法，希望能用来取代结构化编程范型。正如高德纳所构想的那样，

https://leovan.me/cn/2023/03/literate-programming-and-reproducible-research/

**Comment by feynmanma7** [comment]
Great blog!
